### DISCUSSION TITLE
[Saleor 3.0] Multi currency product with variants

### DISCUSSION THREAD

**Discussion by vidhyau-twbs-eu** [saleor/saleor#7934] [Q&A]
We are trying to list product in multiple currency, available at the same time to purchase in the storefront

Here are the configuration in the dashboard
-----------------------------------------------
- Enabled USD and GBP channels
- Product Type with a variant attribute
- Added 2 products 1 with USD and another with GBP.
- Created a product with 2 variants and enabled channels with both  USD and GBP and made it available for listing too 
- Tracking and shipping disabled for both channel products
>>> Images attached below

Questions:

1. Eventhough I set all channels to be shown in the product listing, But still i only see USD channel in the product listing
2. I am unable to see the GBP channel product anywhere to select.

Can anyone guide me how to get the product in both USD and GBP channel listed in the storefront?

 
Dashboard/Product variants
-----------------
![image](https://user-images.githubusercontent.com/88091860/130803545-fd330fc8-8b73-4431-8f0f-7553233d0ef1.png)

Dashboard Channel Manage
------------------------------
![image](https://user-images.githubusercontent.com/88091860/130804437-b85973f5-9fa7-4631-b406-8969de051486.png)

Storefront/Product Listing
-----------------------------
![image](https://user-images.githubusercontent.com/88091860/130804009-ce47091f-72d0-4380-86ba-389d46c1543c.png)

Storefront/Product Selection Page
------------------------------------
![image](https://user-images.githubusercontent.com/88091860/130804145-fd0ff52b-e5cc-4ca0-8a3a-fa1f9cfca2e6.png)

**Comment by karolkielecki** [comment]
You need to create another sales channel with a different currency. For each sales channel, you can assign only one currency. More info:
- https://docs.saleor.io/docs/3.0/developer/channels
- https://docs.saleor.io/docs/3.0/dashboard/channels

**Reply by vidhyau-twbs-eu** [reply]
Thankyou for your answer,

I want to guidance to achieve selling different currency products from a the saleor storefront 

1. Is there a way for me to create 2 different channels, 1 for GBP and another for USD
and access it from a single storefront? a currency switcher 
2. Any documentation on creating multiple shops (linked to its appropriate currency) connected to single dashboard

Thanks for your help.

**Reply by karolkielecki** [reply]
Hey, each channel has only one currency. A currency switcher is not available. Please check the documentation links above.
